src/components/card/card-img.js

```diff
@@ -69,7 +70,7 @@ export const BCardImg = /*#__PURE__*/ Vue.extend({
         class: [baseClass],
         attrs: {
           src: props.src || null,
```

Comment:
@Hiws I think we would be better off if we just pass the value the user provides down to the attrs.

Reply:
I could default the prop to `null`, and then pass the prop directly without any checks if that's what you mean.

Reply:
Yeah, exactly.